pom.xml

```diff
@@ -262,7 +262,7 @@
         <gravitee-entrypoint-http-get.version>1.0.3</gravitee-entrypoint-http-get.version>
         <gravitee-entrypoint-http-post.version>1.0.2</gravitee-entrypoint-http-post.version>
         <gravitee-entrypoint-sse.version>4.0.2</gravitee-entrypoint-sse.version>
-        <gravitee-entrypoint-webhook.version>2.2.0</gravitee-entrypoint-webhook.version>
+        <gravitee-entrypoint-webhook.version>2.2.1-apim-6600-handle-malformed-messages-22x-SNAPSHOT</gravitee-entrypoint-webhook.version>
```

Comment:
Don't forget to replace with the released version

Reply:
Done